File: luck.pas
{ constraints }
const
    MAXK = 200;
    MAXC = 10000;

{ input data }
var
    K, C, i : longint;
    L       : array[0..MAXK-1] of longint;

begin
{
    uncomment the following lines if you want to read/write from files
    assign(input,  'input.txt');  reset(input);
    assign(output, 'output.txt'); rewrite(output);
}

    readln(K, C);
    for i:=0 to K-1 do
        readln(L[i]);

    { insert your code here }

    writeln(42); { change 42 with actual answer }
end.

File: luck.cpp
/*
 * This template is valid both in C and in C++,
 * so you can expand it with code from both languages.
 */

#include <assert.h>
#include <stdio.h>
#include <string>


// constraints
#define MAXK 200
#define MAXC 10000

// input data
int K, C, i;

    using namespace std;

int main() {
  //  uncomment the following lines if you want to read/write from files
  //  freopen("input.txt", "r", stdin);
  //  freopen("output.txt", "w", stdout);

  assert(2 == scanf("%d %d", &K, &C));
  int L[K];
  for (i = 0; i < K; i++){
        assert(1 == scanf("%d", &L[i]));
        if(L[i]%1111==0){
            string a= to_string(L[i]/1000);
            printf("%s\n", string(a,C));
            return 0;
        }
  }





  // insert your code here


  printf("%d\n", 42);  // change 42 with actual answer
  return 0;
}
